core/templates/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.component.html

```diff
@@ -81,7 +81,7 @@ <h3>Load dummy new structures data (only admins)</h3>
       <div class="row">
         <span class="col-lg-8 col-md-8 col-sm-8">
           Loads 2 topics (one of which is empty), a story and three skills (two in a subtopic)
-          attached to the first topic and a question attached to each skill. 3 explorations
+          attached to the first topic and 3 questions attached to each skill. 3 explorations
```

Comment:
There are 5 questions in total, so there should be ```3 questions in the first skill and 1 question in the second and third skills, respectively.```.

Reply:
@Nik-09 done.